packages/react-mutation-mapper/src/component/track/ExonNumTrack.tsx

```diff
@@ -0,0 +1,155 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { observer } from 'mobx-react';
+import { computed, makeObservable } from 'mobx';
+
+import { OncoKbFilterValue } from '../../filter/OncoKbFilter';
+import { DataFilterType } from '../../model/DataFilter';
+import MutationMapperStore from '../../model/MutationMapperStore';
+import { OncoKbTrackTooltip } from './OncoKbTrackTooltip';
+import { default as Track, TrackProps } from './Track';
+import { TrackItemSpec, TrackItemType } from './TrackItem';
+import { Mutation } from 'cbioportal-utils';
+
+import oncoKbImg from '../../images/oncogenic-only.svg';
+import MutationMapper from '../mutationMapper/MutationMapper';
+
+type ExonNumTrackProps = TrackProps & {
+    store: MutationMapperStore<Mutation>;
+};
+
+export function getOncoKbImage() {
+    return <img src={oncoKbImg} alt="OncoKB Oncogenic Symbol" />;
+}
+
+@observer
+export default class ExonNumTrack extends React.Component<
+    ExonNumTrackProps,
+    {}
+> {
+    constructor(props: any) {
+        super(props);
+        makeObservable(this);
+    }
+
+    @computed get exonNumSpecs(): TrackItemSpec[] {
+        const currentTranscriptId = this.props.store.activeTranscript?.result;
+        type ExonDatum = { rank: number; length: number; start: number };
+        const currentTranscript = this.props.store.transcriptsByTranscriptId[
+            currentTranscriptId as string
+        ];
```

Comment:
`currentTransriptId` might be undefined, you should probably do something like this instead:

```suggestion
        const currentTranscript = currentTranscriptId ? this.props.store.transcriptsByTranscriptId[
            currentTranscriptId
        ]: undefined;
```